### PR TITLE
Add race_conditions_SUITE to dynamic_domains

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

File: big_tests/tests/race_conditions_SUITE.erl [hunk 1]
```diff
@@ -159,8 +159,8 @@ delayiq_iq() ->
                      attrs = [{<<"caller_pid">>, BinCallerPid}]},
     escalus_stanza:iq_get(delayiq_ns(), [Payload]).
 
-domain() ->
-    ct:get_config({hosts, mim, domain}).
+host_type() ->
```
Comment (vkatsuba) [inline big_tests/tests/race_conditions_SUITE.erl:162]: Should be used `-import(domain_helper, [host_type/0]).`.

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/73640) / small_tests / 31faf7a9dc08d96493fd6e3cdf4bf8669445ac77
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3283/73640/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3283/73640/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-16_19.47.03/lib.mongooseim.logs/run.2021-09-16_19.47.03/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74382) / small_tests / 3e2094de46e6bb093823a8e95785f6dc4d747e1b
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3283/74382/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3283/74382/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-17_16.24.09/lib.mongooseim.logs/run.2021-09-17_16.24.09/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74385) / small_tests / 889843139725d93ac908ea813260aff9679658a4
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3283/74385/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3283/74385/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-17_16.25.12/lib.mongooseim.logs/run.2021-09-17_16.25.12/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74584) / small_tests / c0604d4345aa14dd355cde6f6461a05a4aa241a2
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3283/74584/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3283/74584/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-20_10.54.03/lib.mongooseim.logs/run.2021-09-20_10.54.04/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74609) / small_tests / dfa9ec2e29f78c70bac1d5d6ca66c7a7e0cf2ac8
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3283/74609/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3283/74609/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-20_11.05.17/lib.mongooseim.logs/run.2021-09-20_11.05.18/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74662) / small_tests / 3dfc9fc64edb4b52eb573577117e5e1527527e70
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3283/74662/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3283/74662/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-20_16.53.15/lib.mongooseim.logs/run.2021-09-20_16.53.16/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good!

**Pull request merged**